src/base/solvers/solveCobraLP.m

```diff
@@ -1228,13 +1228,59 @@
             ILOGcplex.solve();
 
             origStat   = ILOGcplex.Solution.status;
+            stat = origStat;
             if origStat==1
                 f = osense*ILOGcplex.Solution.objval;
                 x = ILOGcplex.Solution.x;
                 w = ILOGcplex.Solution.reducedcost;
-                y = ILOGcplex.Solution.dual;
+                y = ILOGcplex.Solution.dual;                                            
+            elseif origStat == 4
+                %This is likely unbounded, but could be infeasible 
+                %Lets check, by solving an additional LP with a bounded
+                %objective.
+                %Store the original solution
+                Solution = ILOGcplex.Solution;
```

Comment:
I know, but this is only on one particular solver and the return value was 4 (indicating something we don't know, since we only have -1 / 0 / 1 / 2 / 3 as defined return values). So, in this instance (where we have ibm_cplex and a solution status of 4, we know this is unbounded). But in other situations we might not know the reason for this undefined status 4, and someone might simply check for infeasibility and if its not infeasible (i.e. not 0), do something, which then might be wrong. 
I would prefer not to check every single instance for every individual solver but do this in solveCobraLP, and give a correct status.